website/docs/features/onnx/about.md

```diff
@@ -32,9 +64,9 @@ SynapseML now includes a Spark transformer to bring an trained ONNX model to Apa
+    |:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------|
     | feedDict          | Map the ONNX model's expected input node names to the input DataFrame's column names. Make sure the input DataFrame's column schema matches with the corresponding input's shape of the ONNX model. For example, an image classification model may have an input node of shape `[1, 3, 224, 224]` with type Float. It is assumed that the first dimension (1) is the batch size. Then the input DataFrame's corresponding column's type should be `ArrayType(ArrayType(ArrayType(FloatType)))`. | None                                           |
-    | fetchDict         | Map the output DataFrame's column names to the ONNX model's output node names.                                                                                                                                                                                                                                                                                                                                                                                                                  | None                                           |
+    | fetchDict         | Map the output DataFrame's column names to the ONNX model's output node names. Note that if you put outputs that are intermediate in the model, transform will automatically slice at those outputs.                                                                                                                                                                                                                                                                                            | None                                           |
```

Comment:
Maybe instead of the put outputs language we can be more precise about what arameter setting we are talking about here

Reply:
mentioned to look for Slicing section. We shouldn't repeat the info.